.rebase/override/code/extensions/package.json

```diff
@@ -1,5 +0,0 @@
-{
-  "devDependencies": {
-    "esbuild": "^0.14.49"
```

Comment:
It was introduced by https://github.com/che-incubator/che-code/pull/86.
Currently, it looks obsolete as the `esbuild` version [was bumped upstream recently](https://github.com/microsoft/vscode/commit/7e62f31610a3a6932ed83464ce48d64561c828ac) to `0.15.14`.
https://github.com/che-incubator/che-code/pull/86 was partially reverted by https://github.com/che-incubator/che-code/pull/137. Here is the 2'nd part.
@RomanNikitenko we don't need that anymore, right?

Reply:
yes, taking into account
> he esbuild version [was bumped upstream recently](https://github.com/microsoft/vscode/commit/7e62f31610a3a6932ed83464ce48d64561c828ac) to 0.15.14

we can remove it